docs/samples/Microsoft.ML.Samples/Dynamic/ImageClassification/ResnetV2101TransferLearningTrainTestSplit.cs

```diff
@@ -0,0 +1,307 @@
+﻿
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Threading.Tasks;
+using Microsoft.ML;
+using Microsoft.ML.Transforms;
+using static Microsoft.ML.DataOperationsCatalog;
+using System.Linq;
+using Microsoft.ML.Data;
+using System.IO.Compression;
+using System.Threading;
+using System.Net;
+
+namespace Samples.Dynamic
+{
+    public class ResnetV2101TransferLearningTrainTestSplit
+    {
+        public static void Example()
+        {
+            string assetsRelativePath = @"../../../assets";
+            string assetsPath = GetAbsolutePath(assetsRelativePath);
+
+            var outputMlNetModelFilePath = Path.Combine(assetsPath, "outputs",
+                "imageClassifier.zip");
+
+            string imagesDownloadFolderPath = Path.Combine(assetsPath, "inputs",
+                "images");
+
+            //Download the image set and unzip
+            string finalImagesFolderName = DownloadImageSet(
+                imagesDownloadFolderPath);
+
+            string fullImagesetFolderPath = Path.Combine(
+                imagesDownloadFolderPath, finalImagesFolderName);
+
+            try
+            {
+
+                MLContext mlContext = new MLContext(seed: 1);
+
+                //Load all the original images info
+                IEnumerable<ImageData> images = LoadImagesFromDirectory(
```

Comment:
LoadImagesFromDirectory() is a method that is almost always going to be used by developers.
I think we should consider to have this method provided by the framework.
This is also something being requested by customers.